src/lib.rs

```diff
@@ -10,7 +10,7 @@
 //! information on licensing and copyright.
 
 use anyhow::{anyhow, Result as AnyhowResult};
-use base64;
+use base64::{Engine as _, engine::general_purpose::STANDARD as b64_engine};
```

Comment:
What's the purpose of importing `Engine` as `_`? I'm not familiar with this pattern

Reply:
Usually I use this pattern when I need a trait but don't want to name it to avoid name conflicts, but actually in this case we don't need thi